serverless/serverless-using-brokers.adoc

```diff
@@ -0,0 +1,92 @@
+[id="serverless-using-brokers"]
+= Using brokers with Knative Eventing
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-using-brokers
+
+toc::[]
+
+Knative Eventing uses the `default` broker unless otherwise specified. However, this section describes how you can create your own broker.
+
+== Creating a broker for your namespace automatically using namespace annotation
+
+. Annotate your namespace (replace `default` with the desired namespace):
++
+----
+$ oc label namespace default knative-eventing-injection=enabled
```

Comment:
@maschmid what was the deal about injecting this vs. super admin powers on Openshift ? 

Normal users can not do this, right ?

Reply:
Yes, normal users cannot label even their own namespaces by default on OpenShift.

Reply:
https://issues.redhat.com/browse/SRVKE-111

Reply:
I'll update this to say you must be an admin to do this